opensrp-chw-core/src/main/java/org/smartregister/chw/core/custom_views/NavigationMenu.java

```diff
@@ -241,6 +240,20 @@ public void displayToast(Activity activity, String message) {
         }
     }
 
+    private void registerStockReport(Activity activity) {
```

Comment:
Should this be implemented here or in the chw app in the different flavour files?